plugin/vim-tmux-runner.vim

```diff
@@ -260,6 +265,10 @@ function! s:PromptForRunnerToAttach()
       call s:EchoError("No pane specified. Cancelling.")
     endif
   endif
+  else
+    echom s:pane_index
```

Comment:
I'm guessing this was for debugging? Assuming so, can you remove it?

Reply:
Yes, absolutely! I will remove it 👍